tests/AccountTest.php

```diff
@@ -202,13 +206,94 @@ function deleting_a_user_deletes_its_associated_entities()
             'id' => $bio->id,
         ]);
 
-        // $this->dontSeeInDatabase('conferences', [
-        //     'id' => $conference->id,
-        // ]);
+        $this->dontSeeInDatabase('dismissed_conferences', [
+            'user_id' => $user->id,
+            'conference_id' => $dismissedConference->id,
+        ]);
 
         $this->dontSeeInDatabase('favorites', [
             'user_id' => $user->id,
             'conference_id' => $favoriteConference->id,
         ]);
     }
+
+    /** @test */
+    function users_can_dismiss_a_conference()
+    {
+        $user = factory(App\User::class)->create();
```

Comment:
No need for `App\` (here and in several instances below)

Reply:
Gotcha, yeah I wasn't sure what our convention was since there was some of each already here.  I prefer to use the class import as well.